.github/workflows/license-checker.yaml

```diff
@@ -0,0 +1,27 @@
+name: License checker
+
+on:
+  pull_request:
+      branches: [ develop, master ]
+
+jobs:
+  check-license:
+    runs-on: ubuntu-latest
+    steps:
+      # step 1
+      - name: Checkout
+        uses: actions/checkout@v2.4.0
+      # step 2
+      - name: Check License Header
+        uses: apache/skywalking-eyes/header@v0.4.0
+        with:
+          log: info
+          config: .licenserc.yaml
+          mode: check
+      # step 3
+      - name: Check Dependencies' License
+        uses: apache/skywalking-eyes/dependency@v0.4.0
+        with:
+          log: debug
+          config: .licenserc.yaml
+          mode: resolve
```

Comment:
The `resolve` mode always exits with code 0 and it should follow a step to check the git status is clean, otherwise PRs that break the license will not be blocked.

Example:

https://github.com/apache/skywalking/blob/18d24638138ff3c7d9c13cf2b1a71bc15e8ca713/.github/workflows/skywalking.yaml#L81-L86

Reply:
copy that.